src/administrator/components/com_kunena/install/install.script.php

```diff
@@ -17,11 +17,11 @@ class Com_KunenaInstallerScript
 {
 	protected $versions = array(
 		'PHP' => array (
-			'7.0' => '7.0.4',
+			'7.0' => '7.0.3',
 			'5.6' => '5.6.8',
 			'5.5' => '5.5.13',
 			'5.4' => '5.4.13',
-			'0' => '5.4.35' // Preferred version
```

Comment:
I disagree with this changed. Preferred version is the minimum number that the software should run on. The software will throw warnings everytime you try to install on any 5.6, 5.6, and 5.4 version of the software.